_posts/2016-01-06-alle-in-einem-netz-freifunk-in-dachau.markdown

```diff
@@ -0,0 +1,12 @@
+---
+layout: post
+title:  "Alle in einem Netz: Freifunk in Dachau"
+date:   2016-01-06 13:30:00
+categories: freifunkmuc
+---
+
+Der Regionalteil der SZ berichtete in einem [Artikel][artikel] über die wachsende Freifunk München Community in Dachau.
+
+Ein Knotenbetreiber ist Rudolf Gleixner, der in der Sudetenlandstraße seinen Internetanschluss für alle teilt. Neben dem Freifunk Netz wird das WLAN Angebot von den Stadtwerken rund um das Rathaus und im Familienbad dargestellt. Die kommerzielle Variante ist laut Gleixner umständlich.
```

Comment:
Sudetenlandstraße? CRDrei ist doch in der Schleißheimer Straße.

Noch ne Idee zur Formulierung:
"Neben Freifunk werden die WLAN Angebote der Kommunen dargestellt."
"Gleixner findet die Angebote der Kommunen viel zu umständlich."